packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -281,10 +281,10 @@ class IOSDevice extends Device {
       final ProtocolDiscovery diagnosticDiscovery = new ProtocolDiscovery.diagnosticService(
         getLogReader(app: app), portForwarder: portForwarder, hostPort: debuggingOptions.diagnosticPort);
 
-      final Future<Uri> forwardObsUri = observatoryDiscovery.nextUri();
```

Comment:
nit for the future (don't worry about fixing this now since you've already landed it): avoid abbreviations like "Obs" in "forwardObsUri".